erpnext_pk/public/js/employee.js

```diff
@@ -0,0 +1,8 @@
+frappe.ui.form.on('Employee', {
```

Comment:
Is the Employee field available in version 14 without the HR module/app?

Reply:
Currently the code is being written for version 13 and will be modified to work with version 14 as well .. This part of the code will not break whether there is Employee DocType or not

Reply:
Added checks for missing Employee doctype